src/layer/CMakeLists.txt

```diff
@@ -8,8 +8,6 @@ set(CMAKE_FOLDER "${CMAKE_FOLDER}/VulkanLayerSettings")
 add_library(VulkanLayerSettings STATIC)
 add_library(Vulkan::LayerSettings ALIAS VulkanLayerSettings)
 
-lunarg_target_compiler_configurations(VulkanLayerSettings VUL_WERROR)
```

Comment:
This was actually a bug. `lunarg_target_compiler_configurations` didn't work as intended.

Because it only added to the INTERFACE properties.